test/examples/basic/app.env

```diff
@@ -2,3 +2,4 @@ PORT=80
 OTEL_SERVICE_NAME='express-example'
 OTEL_RESOURCE_ATTRIBUTES='deployment.environment=dev'
 OTEL_LOG_LEVEL='DEBUG'
+OTEL_EXPORTER_OTLP_ENDPOINT=http://collector:55681/v1/traces
```

Comment:
Is `55681` still the default for HTTP OTLP or is it `4318` now?

Reply:
It's the default "at the time of writing this PR". I've finished another one that will take care of it https://github.com/signalfx/splunk-otel-js/pull/319